installer.sh

```diff
@@ -380,18 +381,59 @@ if [[ "$TPM_VERSION" -eq "1" ]] ; then
     fi
     make install
     popd # tpm/tpm4720-keylime
+
 elif [[ "$TPM_VERSION" -eq "2" ]] ; then
-    echo
-    echo "=================================================================================="
-    echo $'\t\t\t\tInstall tpm2-tss'
-    echo "=================================================================================="
-   $PACKAGE_MGR -y install $TPM2_SOFTWARE
-    if [[ $? > 0 ]] ; then
-        echo "ERROR: Package(s) failed to install properly!"
-        exit 1
-    fi
-    systemctl enable tpm2-abrmd
-    systemctl start tpm2-abrmd
+    if [[ "$BUILD_TPM2_SOFTWARE" -eq "1" ]] ; then
+        echo "=================================================================================="
+        echo $'\t\t\t\Building tpm2-tss'
+        echo "=================================================================================="
+        # TPM2_TSS
+        git clone $TPM2TSS_GIT tpm2-tss
+        pushd tpm2-tss
+        git checkout $TPM2TSS_VER
+        ./bootstrap
+        ./configure --prefix=/usr
+        make
+        make install
+        popd # tpm
+        echo
+        echo "=================================================================================="
+        echo $'\t\t\t\tBuild and install tpm2-tools'
+        echo "=================================================================================="
+        git clone $TPM2TOOLS_GIT tpm2-tools
+        pushd tpm2-tools
+        git checkout $TPM2TOOLS_VER
+        ./bootstrap
+        ./configure --prefix=/usr/local
+        make
+        make install
+        popd # tpm
+        echo "=================================================================================="
+        echo $'\t\t\t\tBuild and install tpm2-abrmd'
+        echo "=================================================================================="
+        useradd --system --user-group tss
+        git clone https://github.com/tpm2-software/tpm2-abrmd.git tpm2-abrmd
+        pushd tpm2-abrmd
```

Comment:
We need to `git checkout` the correct version of abrmd that is compatible with the TSS used above.  Note that this block of code was commented out and was intended to be used as a guide.  It might need changes to get it working, and definitely won't work on Fedora (since we are not installing the SELinux policy).